recipe/bld.bat

```diff
@@ -19,8 +19,9 @@ if errorlevel 1 exit 1
 python -c "import os; print('_'.join(os.environ['PKG_VERSION'].split('.')[:2]))" > temp.txt
 set /p MAJ_MIN_VER=<temp.txt
 
-:: Remove non version-specific boost headers
-rm %LIBRARY_INC%\boost-%MAJ_MIN_VER%\boost
+:: Install fix-up for a non version-specific boost include
+xcopy /E/Y %LIBRARY_INC%\boost-%MAJ_MIN_VER%\boost\* %LIBRARY_INC%\boost\
+rmdir /s /q %LIBRARY_INC%\boost-%MAJ_MIN_VER%\boost
```

Comment:
It may be that there was a conflict with the existing `boost` directory from `boost-cpp`. I suppose we could have deleted it and then done this move, but your approach works just as well.